### DISCUSSION TITLE
Use self deployed Embedder and Vector DB .

### DISCUSSION THREAD

**Discussion by Preet1234-ux** [langflow-ai/langflow#1862] [Q&A]
I want to use my self deployed Embedder and Vector DB for my RAG Pipeline . Can you please provide me a work around to achieve this in langflow ?

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
